src/views/partials/about/member.hbs

```diff
@@ -1,12 +1,12 @@
 <div class="d-inline-block mr-3 pb-3 cursor-pointer" data-toggle="collapse" data-target="#{{id}}">
-  <img src={{img}} alt={{name}} class="img-fluid rounded-circle" style="height: 160px; width: 160px;">
+  <img src="{{img}}" alt="{{name}}" class="img-fluid" style="height: 160px; width: 160px;">
```

Comment:
Maybe extract inline styles to `class`?

Reply:
Let me make it in a separate PR, we have a couple of these spread out.